### PR TITLE
fix: explicitly set null for the sc

### PR DESCRIPTION
**Pull request by cwyl02**
**What problem does this PR solve?**:
to make both linter and ceph happy

**Which issue(s) does this PR fix?**:

encountered this problem in CI, where the pvc of s3 bucket fails to bound due to storageclass being `""`

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**If the PR adds a version bump, does it add a breaking change in License**:

- [x] No License Change (or NA).

### CODE REVIEW

**Review by cbuto — APPROVED** [review]

File: services/rook-ceph-cluster/1.10.1/defaults/cm.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ data:
         volumeClaimTemplate:
           spec:
             # Use the default storage class configured in cluster.
-            storageClassName: ""
+            storageClassName: null
```
Comment (cbuto) [inline services/rook-ceph-cluster/1.10.1/defaults/cm.yaml:33]:
```suggestion
            storageClassName: ~
```

Comment (takirala) [inline services/rook-ceph-cluster/1.10.1/defaults/cm.yaml:33]:
```suggestion
            # storageClassName: null
```

File: services/rook-ceph-cluster/1.10.1/defaults/cm.yaml [hunk 2]
```diff
@@ -82,7 +82,7 @@ data:
                     requests:
                       storage: 20Gi
                   # Use the default storage class configured in cluster.
-                  storageClassName: ""
+                  storageClassName: null
```
Comment (takirala) [inline services/rook-ceph-cluster/1.10.1/defaults/cm.yaml:85]:
```suggestion
                  # storageClassName: null
```

**Review by takirala — APPROVED** [review]

File: services/rook-ceph-cluster/1.10.1/defaults/cm.yaml [hunk 3]
```diff
@@ -30,7 +30,7 @@ data:
         volumeClaimTemplate:
           spec:
             # Use the default storage class configured in cluster.
-            storageClassName: ""
+            # not setting storageClass to let it fall to environment default
```
Comment (gracedo) [inline services/rook-ceph-cluster/1.10.1/defaults/cm.yaml:33]:
```suggestion
            # not setting storageClass to let it fall to environment default
            # storageClassName: ""
```
Might still be helpful to include the field just commented out

File: services/rook-ceph-cluster/1.10.1/defaults/cm.yaml [hunk 4]
```diff
@@ -82,7 +82,7 @@ data:
                     requests:
                       storage: 20Gi
                   # Use the default storage class configured in cluster.
-                  storageClassName: ""
+                  # not setting storageClass to let it fall to environment default
```
Comment (gracedo) [inline services/rook-ceph-cluster/1.10.1/defaults/cm.yaml:85]:
```suggestion
                  # not setting storageClass to let it fall to environment default
                  # storageClassName: ""
```

**Pull request merged**